gcloud/datastore/test_client.py

```diff
@@ -494,6 +494,48 @@ def test_get_multi_hit_multiple_keys_different_dataset(self):
         with self.assertRaises(ValueError):
             client.get_multi([key1, key2])
 
+    def test_get_multi_hit_multiple_keys_different_dataset_prefixes(self):
+        from gcloud.datastore.key import Key
+
+        DATASET_ID1 = 'DATASET'
+        DATASET_ID2 = 'e~DATASET'
+        DATASET_ID3 = 's~DATASET'
+        KIND = 'Kind'
+        ID1 = 1234
+        ID2 = 2345
+        ID3 = 3456
+
+        # Make sure our IDs are actually different.
```

Comment:
Yes that is modelled after `test_get_multi_hit_multiple_keys_different_dataset` which does the redundant check after setting the variables, so I was trying to stick to the style (glad it's not a project-wide rule :+1: )